packages/components/src/images/icons/stop.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Here's a simplified version that will also look better in Storybook and Finder:
```
<svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
<path id="Vector" d="M25.9574 4H6.04255C5.50083 4 4.9813 4.21071 4.59825 4.58579C4.2152 4.96086 4 5.46957 4 6V26C4 26.5304 4.2152 27.0391 4.59825 27.4142C4.9813 27.7893 5.50083 28 6.04255 28H25.9574C26.4992 28 27.0187 27.7893 27.4017 27.4142C27.7848 27.0391 28 26.5304 28 26V6C28 5.46957 27.7848 4.96086 27.4017 4.58579C27.0187 4.21071 26.4992 4 25.9574 4Z" />
</svg>
```

Reply:
70f0c624a376fc25afb6a1bcaf986836cb46e00a